Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpBoneImporter.cpp

```diff
@@ -73,31 +73,23 @@ namespace AZ
                 bool isBone = FindFirstBoneByNodeName(currentNode, boneByNameMap);
                 if (!isBone)
                 {
+                    // In case the bone is not listed in any of the meshes in the scene, populate a set of animated bone names from
+                    // the animations. This set of names will help varify if the current node is a bone.
+                    AZStd::unordered_set<AZStd::string> animNodeByNameMap;
```

Comment:
This is basically for being able to animate nodes that are not skinned (and thus a bone), correct?

If yes, maybe we can rename this map to: animatedNodesMap? Does this only contain unskinned nodes or all animated nodes? (so does it include the animated bones?)

Reply:
correct. changed.